### PR TITLE
Info.plist translations

### LINKED ISSUE #458: InfoPlist.strings is not parsed correctly
The InfoPlist.strings file is being translated, but iOS cannot detect the translations.

The Apple developer page proves that our convertTranslations.js file does not handle correctly the InfoPlist.strings.

In [this page](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/AboutInformationPropertyListFiles.html), the InfoPlist.strings format should be:

NSCameraUsageDescription = "Allowing access to the camera lets you take photos and videos."
The key must not contain quote marks. The translations must be quoted.
In this case, iOS cannot detects our files as translated.

### PR DESCRIPTION
**Pull request by OzancanKaratas**
Fixes #458.

The translations was redesigned according to the Apple instructions. untranslated.xml must be imported to Android source.

### CODE REVIEW

File: deltachat-ios/en.lproj/InfoPlist.strings [hunk 1]
```diff
@@ -1,6 +1,4 @@
-"Privacy - Camera Usage Description" = "Allowing access to the camera lets you take photos and videos.";
-"Privacy - Contacts Usage Description" = "Allowing access to your address book lets you chat with contacts from your device.";
-"Privacy - Microphone Usage Description" = "Allowing access to the microphone lets you record audio.";
-"Privacy - Photo Library Usage Description" = "Allowing access to the photo library allows you to upload images from it.";
-"Privacy - Microphone Usage Description" = "Allowing access to the microphone lets you record audio.";
-"Privacy - Photo Library Usage Description" = "Allowing access to the photo library allows you to upload images from it.";
+NSCameraUsageDescription = "Allowing access to the camera lets you take photos and videos.";
```
Comment (cyBerta) [inline deltachat-ios/en.lproj/InfoPlist.strings]: The keys e.g. NSCameraUsageDescription need to be double quoted.

Comment (cyBerta) [reply deltachat-ios/en.lproj/InfoPlist.strings]: whoops. nevermind.

File: deltachat-ios/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -615,10 +615,4 @@
 "a11y_delivery_status_invalid" = "Invalid delivery status";
 "a11y_remove_account_btn_label" = "Remove account";
 "a11y_message_context_menu_btn_label" = "Message actions";
-"a11y_background_preview_label" = "Background preview";
-
-// Translators: These strings are only used on iOS
```
Comment (cyBerta) [inline deltachat-ios/en.lproj/Localizable.strings]: Why did you remove the follwoing lines? 
import_contacts and import_contact_message are still in use.

`no_account_setup` is unused though.

File: tools/convertTranslations.js [hunk 3]
```diff
@@ -161,8 +161,8 @@ function toInfoPlistStrings(lines) {
         if (!key.startsWith("INFOPLIST.")) {
           continue;
         }
-        key = key.replace('INFOPLIST.', '').replace(/\./gi, ' ').replace(/\_/gi, '-');
-        out += `"${key}" = "${line[1]}";\n`;
+        key = key.replace('INFOPLIST.', '');
```
Comment (cyBerta) [inline tools/convertTranslations.js]: nice :), cool that you updated the script as well!

File: tools/untranslated.xml [hunk 4]
```diff
@@ -3,11 +3,9 @@
 
     <!-- iOS specific untranslated strings -->
     <string name="import_contacts">Import device contacts</string>
```
Comment (cyBerta) [inline tools/untranslated.xml]: import_contacts can be deleted as well after the string was added to the android repository - like the line below (import_contacts_message).

**Comment by cyBerta** [comment]
nice! thanks for fixing that! If you undo the deletion of import_contacts and import_contacts_message from deltachat-ios/en.lproj/Localizable.strings this PR is ready to merge

File: deltachat-ios/en.lproj/Localizable.strings (hunk 2 above)
Comment (cyBerta) [reply deltachat-ios/en.lproj/Localizable.strings]: import_contacts and import_contact_message cannot be deleted. They are referenced in the project. 
The same keys exist in untranslated.xml but that is only the source for pulling strings into this file. untranslated.xml is not referenced in the project!

Comment (OzancanKaratas) [reply deltachat-ios/en.lproj/Localizable.strings]: I understand what you mean. However, when we import the entries in the untranslated.xml file into our Android project, the strings I have already deleted will come back. Please merge this.

**Comment by OzancanKaratas** [comment]
@r10s, please review and comment. I feel that the Contacts text is bad.

File: deltachat-ios/Info.plist [hunk 5]
```diff
@@ -39,13 +39,13 @@
...
 	<true/>
 	<key>NSCameraUsageDescription</key>
-	<string>Allowing access to the camera lets you take photos and videos.</string>
+	<string>This is used to take photos and videos during chats or scan QR codes.</string>
 	<key>NSContactsUsageDescription</key>
-	<string>Allowing access to your address book lets you chat with contacts from your device.</string>
+	<string>This is used to import email addresses of your contacts.</string>
 	<key>NSMicrophoneUsageDescription</key>
-	<string>Allowing access to the microphone lets you record audio.</string>
+	<string>This is used to record voice messages during chats.</string>
```
Comment (r10s) [inline deltachat-ios/Info.plist]: i find this "during chats" wording a bit strage. may be regarded as if this always happens "during chatting".
maybe better sth. as "Delta Chat needs access to your microphone to record voice messages you can send to your chat contacts."

Comment (OzancanKaratas) [reply deltachat-ios/Info.plist]: But aren't we using the microphone to send voice messages only in chats?

Comment (r10s) [reply deltachat-ios/Info.plist]: sure, but this "during chats" wording sounds strange to me - and may sound strange to reviewers as well. maybe i am wrong. 

for my suggestion, i peeked on signal wordings, btw, i think this is not the worse idea as they make the review process at some point. maybe just take the sentences from https://github.com/signalapp/Signal-iOS/blob/master/Signal/Signal-Info.plist#L116 and adapt them to our usage.

Comment (OzancanKaratas) [reply deltachat-ios/Info.plist]: I have written explanations that are more general, but more understandable by the user. We can change these comments as we release new features. For now, at least I couldn't find anything better for the microphone.
